ACPs/75-acceptance-proofs/README.md

```diff
@@ -111,23 +74,26 @@ The `GetAcceptanceSignatureRequest` message is sent to a peer to request their s
 #### Gossip
 ```diff
 + message GetAcceptanceProofRequest {
-+   bytes block_id = 1;
++   bytes chain_id = 1;
++   bytes block_id = 2;
```

Comment:
As talked about offline, we should be requesting the proof by `height` not by `block_id`.